examples/llm_ptq/hf_ptq.py

```diff
@@ -707,13 +696,6 @@ def output_decode(generated_ids, input_shape):
         choices=KV_QUANT_CFG_CHOICES.keys(),
         help="Specify KV cache quantization format, default to fp8 if not provided",
     )
-    parser.add_argument(
-        "--export_fmt",
-        required=False,
-        default="tensorrt_llm",
-        choices=["tensorrt_llm", "hf"],
-        help=("Checkpoint export format"),
-    )
```

Comment:
For this top-level API, I'd suggest we still keep `--export_fmt` argument but ignore it and issue a warning that its deprecated and we remove the argument one release later